src/plugins/memoryvalue/memoryvalue.c

```diff
@@ -0,0 +1,188 @@
+/**
+ * @file
+ *
+ * @brief memoryvalue plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#define _XOPEN_SOURCE
+#include "memoryvalue.h"
+#include <kdberrors.h>
+#include <kdbhelper.h>
+#include <kdbtypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+static char * deblank (char * input)
```

Comment:
Why are you returning the pointer if you modify the content of input itself? Please add a short docu what this function is doing.